src/transformers/models/detr/configuration_detr.py

```diff
@@ -155,6 +155,10 @@ def __init__(
         bbox_loss_coefficient=5,
         giou_loss_coefficient=2,
         eos_coefficient=0.1,
+        in_chans=3,
```

Comment:
```suggestion
        num_channels=3,
```
I would rename this to num_channels to be consistent with other models in the library (like ViT).